### PR TITLE
Updated README.md (added support resources)

### PR DESCRIPTION
**Pull request by LenzGr**
Updated README.md and added a "Support" section, referring to the Wiki and mailing list.

### CODE REVIEW

**Comment by theanalyst** [comment]
Maybe Get Involved instead of support?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 ## Status
 Automatic discovery, configuration and deployment of Ceph clusters works. RGW
 deployment is currently broken. MDS deployment and CephFS creation works.
 
+## Support
+To learn more about DeepSea, take a look at the [Wiki](https://github.com/SUSE/DeepSea/wiki).
+
+There is also a dedicated mailing list [deepsea-users](http://lists.suse.com/mailman/listinfo/deepsea-users).
+If you have any questions, suggestions for improvements or any other
+feedback, please joins us there! We look forward to your contributions.
```
Comment (theanalyst) [inline README.md]: We could add a note on issues, currently github itself, https://github.com/SUSE/DeepSea/issues/new too

Comment (theanalyst) [inline README.md]: Also s/joins/join

Comment (theanalyst) [inline README.md]: And maybe just contribution instead of contributions (just being pedantic here)

Comment (LenzGr) [inline README.md]: Thanks for the comments, I'll revise my PR shortly.

**Comment by rjfd** [comment]
I'm going to be the Git Cop here :-) 
The two commits in this PR should have been squashed in a single commit to maintain the history organization.
Let's try to make an effort to have the log organized.

**Comment by smithfarm** [comment]
> The two commits in this PR should have been squashed in a single commit to maintain the history organization.

@rjfd is right +1

**Comment by LenzGr** [comment]
Thanks, I'll keep that in mind for the future - I actually just used the GitHub web frontend to make these amendments :)

**Pull request merged**